ui/static/ui/js/learning_resources.jsx

```diff
@@ -135,12 +137,27 @@ define('learning_resources', [
         thiz.setState({
           message: "Form saved successfully!"
         });
+        thiz.hideLoaderAndScrollUp();
       }).fail(function () {
         thiz.setState({
           message: {error: "Unable to save form"}
         });
+        thiz.hideLoaderAndScrollUp();
       });
     },
+    hideLoaderAndScrollUp: function() {
+      var scrollableDiv = $(this.getDOMNode()).closest(
```

Comment:
React components should not touch anything outside of the component hierarchy. You should not read from the DOM in a React component and you certainly shouldn't touch any of a DOM element's ancestors.

If you need a React component to make a modification outside the component you should pass it a callback in the props and execute that. That callback should not access React's DOM either, but here you shouldn't need to.